source/jormungandr/jormungandr/utils.py

```diff
@@ -376,6 +378,57 @@ def generate_id():
     return shortuuid.uuid()
 
 
+def get_pt_object_from_json(dict_pt_object):
+    if not isinstance(dict_pt_object, dict):
+        logging.getLogger(__name__).error('Invalid dict_pt_object')
+        raise InvalidArguments('Invalid dict_pt_object')
+    map_pt_object_type = {
+        "stop_point": type_pb2.STOP_POINT,
+        "stop_area": type_pb2.STOP_AREA,
+        "address": type_pb2.ADDRESS,
+        "administrative_region": type_pb2.ADMINISTRATIVE_REGION,
+        "poi": type_pb2.POI,
+        "access_point": type_pb2.ACCESS_POINT,
+    }
```

Comment:
This doesn't need to be instantiated on each call of the function, it can be a global `const`, isn't it?

Reply:
Done